MIGRATION.md

```diff
@@ -310,6 +311,55 @@
 
 ## From version 7.5.0 to 7.6.0
 
+#### Using implicit actions during rendering is deprecated (for example in the play function)
```

Comment:
I would remove (for example in the play function) to shorten the heading. You can explain in the contents.

Reply:
Sounds good